app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::BooksController < ApplicationController
+  def index
+    # @books = Book.all
```

Comment:
As a best practice, we do try to avoid merging commented-out code into the `master` branch.